packages/lesswrong/server/migrations/20250503T040404.add_llmEvaluations_and_index.ts

```diff
@@ -0,0 +1,15 @@
+import AutomatedContentEvaluations from "../collections/automatedContentEvaluations/collection";
+import { createTable, dropTable, updateIndexes } from "./meta/utils";
+
+export const up = async ({db}: MigrationContext) => {
+  // There is not much data in this table, and it's not too important.
```

Comment:
(One way to do this without dropping the table would be to manually create the columns as nullable, populate the existing records with some sane values, and then alter the table to set the columns to be non-nullable.  But it is more annoying.)